packages/eslint-plugin/rules/data-no-store-string-literals.js

```diff
@@ -1,3 +1,33 @@
+/**
+ * Converts store name to variable name.
+ * Removes dashes and uppercases the characters after dashes and appends `Store` at the end.
+ *
+ * @param {string} storeName
+ * @return {string} store name as variable name
+ */
+function storeNameToVariableNames( storeName ) {
+	return (
+		storeName
+			.split( '-' )
+			.map( ( s, index ) =>
```

Comment:
Nitpicking: let's avoid ambiguous name here:

```suggestion
			.map( ( value, index ) =>
```